### ISSUE TITLE
Add clarity for how to run scripts

### ISSUE THREAD

**Issue by juanshishido** [twitterdev/ads-platform-tools#14]
I think we could make the usage instructions in `README.md` clearer.

For Python, the current example is:

```
./hash_tailored_audience_file.py --type EMAIL --infile /data/source_email_list.txt --outfile /data/hashed_email_list.txt
```

Several points of confusion:

1. `./hash_tailored_audience_file.py` implies we are in `ads-platform-tools/python/`&mdash;better to be explicit
2. the `--infile` and `--outfile` args use absolute paths, pointing to `/data/`, a branch off the root directory

---

One idea is for the examples to show the scripts being run from `ads-platform-tools/`. To the repository, we would add `data/` and include the input file as follows:

```
├── LICENSE
├── README.md
├── data
│   └──input_email.txt
├── perl
│   └── hash_tailored_audience_file.pl
├── python
│   ├── compare_stats.py
│   ├── fetch_stats.py
│   ├── hash_mact_device.py
│   ├── hash_tailored_audience_file.py
│   ├── mact_keys.yml
│   ├── requirements.pip
│   └── setup.cfg
├── ruby
│   └── sample_pmfi.rb
└── screenshot.png
```

The Python example would then be:

```
~/ads-platform-tools $ python python/hash_tailored_audience_file.py --type EMAIL --infile data/input_email.txt --outfile data/output_email.txt
```

Thoughts?

**Comment by juanshishido** [comment]
cc: @jaakko-sf, @andypiper

**Comment by andypiper** [comment]
LGTM @juanshishido #shipit

**Comment by juanshishido** [comment]
closed by 49ab358aa94979e2e7b43cd8b755714aaf2de52e
